cves/kernel/CVE-2021-26708.yml

```diff
@@ -456,7 +466,8 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    This seems to be a coding mistake, the developer made an assignment before it was ready. This created the vulnerability.
```

Comment:
I feel like this area should be expanded a bit. It doesn't really provide any interesting details about the mistake other than stating it.